tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.AspNet.Server.Kestrel.LibuvCopier
+{
+    public class Program
+    {
+        public void Main(string[] args)
+        {
+            try
+            {
+                var lockJson = JObject.Parse(File.ReadAllText("project.lock.json"));
+                var dnxHome = Environment.GetEnvironmentVariable("DNX_HOME");
+
+                foreach (var libuvLib in lockJson["libraries"].OfType<JProperty>().Where(
+                    p => p.Name.StartsWith("Microsoft.AspNet.Internal.libuv", StringComparison.Ordinal)))
+                {
+                    foreach (var filePath in libuvLib.Value["files"].Select(v => v.Value<string>()))
```

Comment:
Yes. I copy them and create the structure that will work with native dll loading.